dev-qt/qtnetworkauth/qtnetworkauth-5.9999.ebuild

```diff
@@ -0,0 +1,23 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit qt5-build
+
+DESCRIPTION="Network authorization component library for the Qt5 framework"
+LICENSE="GPL-3"
```

Comment:
Sorry, could you let me know more about your question?
It's no problem if you concern the license. qtnetworkauth is published under GPL-3 [1]

[1] http://code.qt.io/cgit/qt/qtnetworkauth.git/tree/?h=5.9